lib/gtl_formatter.rb

```diff
@@ -32,6 +32,7 @@ class GtlFormatter
     'archived'                  => 'fa fa-archive',
     'orphaned'                  => 'ff ff-orphaned',
     'retired'                   => 'fa fa-clock-o',
+    'non_operational'           => 'pficon pficon-exclamation',
```

Comment:
@agrare icon-wise should this be matched to something that already exists?  I see `pficon-unplugged` below, perhaps.  I'm fine as is, I just noticed this introduced a "new" icon, so raising the quetsion.

Reply:
The `raw_power_state` that this is mapped to is `ERROR` so I think exclamation or some other error indication would be better than just unplugged.